arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,881 @@
+    "AccountResourceGroupGetResponse": {
+      "description": "The response to an account resource group GET request.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of resource details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccountResourceGetResponse"
+          }
+        }
+      }
+    },
+    "AccountResourcePutRequestData": {
+      "description": "The body of a PUT request to modify a Visual Studio account resource.",
+      "type": "object",
+      "properties": {
+        "accountName": {
+          "description": "Gets the account name",
+          "type": "string"
+        },
+        "email": {
+          "description": "Gets the email.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "location": {
+          "description": "Gets or sets the Azure instance location.",
+          "type": "string"
+        },
+        "operationType": {
+          "description": "Gets the type of the operation.",
+          "enum": [
+            "unknown",
+            "create",
+            "update",
+            "link"
+          ],
+          "x-ms-enum": {
+            "name": "Microsoft.VisualStudio.Services.Commerce.AccountResourcePutRequestOperationType"
+          }
+        },
+        "properties": {
+          "description": "Gets or sets the custom properties of the resource.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-client-flatten": true
+        },
+        "resourceGroupName": {
+          "description": "Gets the name of the resource group.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "resourceName": {
+          "description": "Gets the name of the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "resourceProviderNamespace": {
+          "description": "Gets the resource provider namespace.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "resourceType": {
+          "description": "Gets the type of the resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "subscriptionId": {
+          "description": "Gets the subscription identifier.",
+          "readOnly": true,
+          "type": "string",
+          "format": "uuid"
+        },
+        "tags": {
+          "description": "Gets or sets the custom tags of the resource.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tenantId": {
+          "description": "Gets the tenant identifier.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "upn": {
+          "description": "Gets the upn.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "ExtensionResourceGetResponse": {
```

Comment:
Given we agreed on `AccountResource` (or `AccountResourceResult`), I think this should be `ExtensionResource` right?